runtime/statemine/src/lib.rs

```diff
@@ -793,8 +819,14 @@ impl_runtime_apis! {
 			let params = (&config, &whitelist);
 
 			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
-			add_benchmark!(params, batches, pallet_balances, Balances);
-			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
+			//TODO we should re run all benchmarks for used pallets together
```

Comment:
should be fixed IMO

Reply:
ya, but I was thinking it falls out of scope of this PR and to make an issue and do it after this is merged

Reply:
created issue #37